README.md

```diff
@@ -85,13 +85,15 @@ run without likelihood, set the variable `self.useLikelihood` in
 [numenta_detector.py](https://github.com/numenta/NAB/blob/master/nab/detectors/numenta/numenta_detector.py)
 to `False`.
 * [nab-comportex](https://github.com/floybix/nab-comportex) is a twist on HTM  anomaly detection using [Comportex](https://github.com/htm-community/comportex), a community-driven HTM implementation in Clojure. Please see [Felix Andrew's blog post](http://floybix.github.io/2016/07/01/attempting-nab) on experiments with this algorithm.
+* [HTM Java](https://github.com/numenta/htm.java) is a Community-Driven Java port of HTM.
 
 
 | Detector      |Standard Profile | Reward Low FP | Reward Low FN |
 |---------------|---------|------------------|---------------|---------------|
 | Numenta HTM*   | 65.3             | 58.6       | 69.4          |
 | [nab-comportex](https://github.com/floybix/nab-comportex)&dagger; | 64.6             | 58.8       | 69.6          |
 | [NumentaTM HTM](https://github.com/numenta/NAB/blob/master/nab/detectors/numenta/numentaTM_detector.py)* | 61.2             | 52.4       | 66.1          |
+| [HTM Java](https://github.com/numenta/NAB/blob/master/nab/detectors/htmjava) | 57.54 | 49.71 | 62.33 |
```

Comment:
Guys,

Can we please get HTM.Java's results = NuPIC before we post? Please take this down? There's no reason why HTM.Java's results have to be less than NuPIC, so why would we post these without optimizing it?